src/transactions/transferFeedUtils.ts

```diff
@@ -400,3 +426,56 @@ export function isTransferTransaction(
 ): tx is TransferItemFragment {
   return (tx as TransferItemFragment).address !== undefined
 }
+
+// Note: This hook is tested from src/transactions/feed/TransferFeedItem.test.ts
+function useFiatConnectTransferDisplayInfo({ amount, transactionHash }: TokenTransfer) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(amount.tokenAddress)
+  const fcTransferDetails = useSelector(getCachedFiatConnectTransferSelector(transactionHash))
+  const cachedFiatAccountUses = useSelector(cachedFiatAccountUsesSelector)
+  const account = useMemo(
+    () =>
+      fcTransferDetails?.quote.transferType === TransferType.TransferOut
+        ? cachedFiatAccountUses.find(
+            ({ fiatAccountId }) => fiatAccountId === fcTransferDetails.fiatAccountId
+          )
+        : undefined,
+    [cachedFiatAccountUses, fcTransferDetails]
+  )
+  if (!account || !fcTransferDetails) {
```

Comment:
```suggestion
  if (!account || !fcTransferDetails || (fcTransferDetails?.quote.transferType !== TransferType.TransferOut)) {
```
this is one thing you could do to deal with transfers in for now. I'm less concerned about efficiency than I am about simplicity and maintainability